offchain/rollups-events/tests/tls.rs

```diff
@@ -0,0 +1,26 @@
+use std::process::Command;
+use test_fixtures::broker_redis::BrokerRedisFixture;
+use testcontainers::clients::Cli;
+
+/// Currently, the redis client we're using does not provide a high-level API to
+/// load certificates, it only loads the ones in the host machine's certificate
+/// store. To avoid having to install certificates in the machine running the
+/// test, it builds and runs a Docker image containing both the Redis server
+/// and a test binary that connects to it, executing the latter after the
+/// container is up.
+#[test_log::test(tokio::test)]
+async fn test_it_connects_via_tls() {
+    let docker = Cli::default();
+    let fixture = BrokerRedisFixture::setup(&docker);
+    let output = Command::new("docker")
```

Comment:
This could be a command in the `docker_cli` module.

Reply:
I left it here so I could assert the exit code, which otherwise would be encapsulated within `docker_cli::docker_run()`. Since this is the only place we're doing this, I didn't feel the need to create a new function in `docker_cli`.

But I can move it there, no problem. Should I keep asserting the exit code (thus making it slightly different from the other functions in the module) or should I assert std_out?

Reply:
I didn't quite understand. the `docker_cli::docker_run` function asserts the exit code like you do here.

Reply:
Sorry, I explained myself poorly. I was just asking about the preferred function signature for `docker_cli::exec`: if it would be better to return `std_out` (the return from `docker_cli::docker_run`) or make it a bit different from the other functions and return `std::process::Output`. 

I'll follow the pattern and just return std_out 😉